deploy/firesim

```diff
@@ -257,7 +257,7 @@ def buildbitstream(build_config_file: BuildConfigFile) -> None:
         execute(build_config.bitbuilder.replace_rtl, hosts=['localhost'])
         execute(build_config.bitbuilder.build_driver, hosts=['localhost'])
 
-    def release_build_hosts_handler(sig, frame) -> None:
+    def release_build_hosts_handler(sig: Any, frame: Any) -> None:
```

Comment:
I'm assuming there's no way to provide a more specific type annotation here?

Reply:
Since there are alot of `Any`s left throughout the codebase, I figured this could just be done in a later PR / cleanup pass.

Reply:
```suggestion
    def release_build_hosts_handler(sig: signal.Signals, frame: types.FrameType) -> None:
```
should work.